lib/Dialect/FIRRTL/Transforms/InferWidths.cpp

```diff
@@ -2099,7 +2099,7 @@ bool InferenceTypeUpdate::updateOperation(Operation *op) {
 /// Resize a `uint`, `sint`, or `analog` type to a specific width.
 static FIRRTLBaseType resizeType(FIRRTLBaseType type, uint32_t newWidth) {
   auto *context = type.getContext();
-  return TypeSwitch<FIRRTLBaseType, FIRRTLBaseType>(type)
+  return TypeSwitch<FIRRTLBaseType, FIRRTLBaseType>(type.getAnonymousType())
```

Comment:
Looks like width inference also drops type aliases?

Please add at least basic tests for these passes to cover the new functionality.